### PR TITLE
chore/update to 2.0.0-rc.3 app version

### LINKED ISSUE #172: Update Flux to v2.0.0-rc.2
Please update Flux to v2.0.0-rc.2, the change log is here:
- RC.2 https://github.com/fluxcd/flux2/releases/tag/v2.0.0-rc.2
- RC.1 https://github.com/fluxcd/flux2/releases/tag/v2.0.0-rc.1

### PR DESCRIPTION
**Pull request by jimid27**
#### What this PR does / why we need it:

This PR (could be a temporary item) updates the flux manifests/CRDs to use the latest version of flux (2.0.0-rc.3) for those who want to adopt the use of the Helm charts to bootstrap the cluster on the new version.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #172 

#### Special notes for your reviewer:
- Tried `make generate` and it sort of worked but kept added "U" as part of the CRDs.  Not exactly sure why that happened at this point.
#### Checklist

- [X] [DCO](https://github.com/fluxcd-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] helm-docs are updated
- [X] Helm chart is tested
- [X] Update artifacthub.io/changes in Chart.yaml
- [X] Run `make reviewable`

### CODE REVIEW

File: charts/flux2-notification/Chart.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: flux2-notification
 type: application
 version: 1.8.0
```
Comment (stefanprodan) [inline charts/flux2-notification/Chart.yaml:4]: Can you please bump the version here and the APIs in this chart to `notification.toolkit.fluxcd.io/v1beta2`

Comment (jimid27) [reply charts/flux2-notification/Chart.yaml]: Bumped - sorry about that

File: charts/flux2-sync/templates/flux-kustomization.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.kustomization }}
-apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+apiVersion: kustomize.toolkit.fluxcd.io/v1
 kind: Kustomization
```
Comment (stefanprodan) [inline charts/flux2-sync/templates/flux-kustomization.yaml:3]: Please bump the apiVersion to v1 in the kustomizationlist below. Thanks

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @jimid27 for saving the day, I was about to archive this repo.

**Comment by jimid27** [comment]
> LGTM
> 
> Thanks @jimid27 for saving the day, I was about to archive this repo.

No problem! Looking forward to using these charts!

**Pull request merged**